docs/whats-new/dotnet-docs-mod1.md

```diff
@@ -1,150 +1,139 @@
-- [jjolidon](https://github.com/jjolidon) -  ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [jmangel](https://github.com/jmangel) -  ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [KUTlime](https://github.com/KUTlime) - Radek Zahradník ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [kyanha](https://github.com/kyanha) - Kyle H ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [mavaddat](https://github.com/mavaddat) - Mavaddat Javid ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [mpidash](https://github.com/mpidash) - Mario Pistrich ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [nnpcYvIVl](https://github.com/nnpcYvIVl) -  ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [owre](https://github.com/owre) - Pontus Öwre ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [postmeback](https://github.com/postmeback) - Arka Poddar ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [Raienryuu](https://github.com/Raienryuu) - Piotr ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [roboz0r](https://github.com/roboz0r) -  ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [sa-es-ir](https://github.com/sa-es-ir) - Saeed Esmaeelinejad ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [Sergey-Vlasov](https://github.com/Sergey-Vlasov) - Sergey Vlasov ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [sliekens](https://github.com/sliekens) - Steven ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [smoothdeveloper](https://github.com/smoothdeveloper) - Gauthier Segay ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [stesmi](https://github.com/stesmi) - Stefan Smietanowski ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [sungam3r](https://github.com/sungam3r) - Ivan Maximov ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [SusheelThapa](https://github.com/SusheelThapa) - Susheel Thapa ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
-- [Tchial0](https://github.com/Tchial0) - Chialo Armando ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [chaojian-zhang](https://github.com/chaojian-zhang) - Charles Zhang ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
+- [dotnet-policy-service](https://github.com/dotnet-policy-service) -  ![1 pull requests.](https://img.shields.io/badge/Merged%20Pull%20Requests-1-green)
```

Comment:
We should remove a bot:

```suggestion
```

Reply:
We should perhaps add, `dotnet-policy-service` and the `learn-build-service-prod` (as found [here](https://github.com/dotnet/docs-aspire/pull/344/commits/8ed38c91ef438061c835a62e1fc1930fefc1bf36)) to the exclusion list.